android/KMEA/app/src/main/assets/android-host.js

```diff
@@ -168,18 +205,13 @@ function enableSuggestions(model, mayPredict, mayCorrect) {
   keyman.core.languageProcessor.mayPredict = mayPredict;
   keyman.core.languageProcessor.mayCorrect = mayCorrect;
 
-  registerModel(model);
+  keyman.addModel(model);
 }
 
 function setBannerOptions(mayPredict) {
   keyman.core.languageProcessor.mayPredict = mayPredict;
 }
 
-function registerModel(model) {
-  //window.console.log('registerModel: ' + model);
-  keyman.addModel(model);
-}
-
```

Comment:
why is this change in this PR?

Reply:
I originally started banner management by keeping similarity with ios-host.js

https://github.com/keymanapp/keyman/blob/8ebb7257b3dcef7344a6d9b753ce3d53fb8ca4f7/ios/engine/KMEI/KeymanEngine/resources/Keyman.bundle/Contents/Resources/ios-host.js#L323-L330

Reply:
reverted